test/acceptance/testing.go

```diff
@@ -87,12 +88,43 @@ func (c *AccTestCase) getResult(t *testing.T) *ScanResult {
 	return NewScanResult(t, analysis)
 }
 
+/**
+ * Retrieve env from os.Environ() but override every variable prefixed with ACC_
+ * e.g. ACC_AWS_PROFILE will override AWS_PROFILE
+ */
+func (c *AccTestCase) resolveTerraformEnv() []string {
+
+	environMap := make(map[string]string, len(os.Environ()))
+
+	const PREFIX string = "ACC_"
+
+	for _, e := range os.Environ() {
+		envKeyValue := strings.SplitN(e, "=", 2)
+		if strings.HasPrefix(envKeyValue[0], PREFIX) {
+			varName := strings.TrimPrefix(envKeyValue[0], PREFIX)
+			environMap[varName] = envKeyValue[1]
+			continue
+		}
+		if environMap[envKeyValue[0]] == "" {
+			environMap[envKeyValue[0]] = envKeyValue[1]
+		}
+	}
```

Comment:
I think there is one case that it will not be overridden but I don't know if it's possible to be in that special case. Imagine this one in that exact order:

```json
os.Setenv("ACC_TEST_VAR_5", "")
os.Setenv("TEST_VAR_5", "test2")
```

The output would be that `TEST_VAR_5` at the end will be `test2`.

Reply:
Goot catch, my test to check if key is present in map was weak, fixed using this form 👍🏻 

```go
		if _, alreadyExist := environMap[envKeyValue[0]]; !alreadyExist {
			environMap[envKeyValue[0]] = envKeyValue[1]
		}
```

Test updated